webapi/helpers.go

```diff
@@ -141,3 +148,59 @@ func isValidTicket(tx *wire.MsgTx) error {
 
 	return nil
 }
+
+// validateTicketHash ensures the provided ticket hash is a valid ticket hash.
+// A ticket hash should be 64 chars (MaxHashStringSize) and should parse into
+// a chainhash.Hash without error.
+func validateTicketHash(c *gin.Context, hash string) (bool, error) {
```

Comment:
This func doesn't need to return a bool. The bool doesn't give any extra information versus just checking `if err != nil` (it actually gives less, because there is no descriptive error message).

See `isValidTicket` for a similar pattern where only an error is returned.